java/connectors/semantickernel-connectors-memory-jdbc/src/main/java/com/microsoft/semantickernel/connectors/memory/jdbc/Connector.java

```diff
@@ -0,0 +1,44 @@
+package com.microsoft.semantickernel.connectors.memory.jdbc;
+
+import com.microsoft.semantickernel.memory.DataEntryBase;
+import reactor.core.publisher.Mono;
+
+import java.sql.Connection;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+public interface Connector {
```

Comment:
Is it possible to have a design review of this JDBC work? JDBC implies compliance to a specific set of interfaces and I could very well be wrong here but this appears to make use of JDBC but isn't JDBC on it's own as it doesn't comply to the interfaces.